### PR TITLE
Fix compilation with latest ctrulib

### PR DESCRIPTION
**Pull request by FenrisulfrX**

### CODE REVIEW

**Comment by archshift** [comment]
If you're doing this you might as well remove the rest of the boilerplate inits which are made redundant by calling `gfxInitDefault()`.

**Comment by FenrisulfrX** [comment]
@archshift Done.

**Comment by archshift** [comment]
InitOutput and DeinitOutput are our functions; they still need to be there. Have you tested to see if this works?

**Comment by FenrisulfrX** [comment]
I tested on Citra and it worked, however you are right, testing it on hardware made it crash.

**Comment by archshift** [comment]
LGTM

File: source/main.cpp [hunk 1]
```diff
@@ -49,11 +44,8 @@ int main(int argc, char** argv)
 
     ClearScreens();
     
-    DeinitOutput();
-    fsExit();
     gfxExit();
-    hidExit();
-    aptExit();
-    srvExit();
+    DeinitOutput();
```
Comment (yuriks) [inline source/main.cpp]: Looks like you switched the order of these call. Is that intended?

Comment (bunnei) [inline source/main.cpp]: It doesn't matter either way

**Comment by bunnei** [comment]
LGTM

**Pull request merged**